src/main/java/org/jabref/logic/cleanup/Cleanups.java

```diff
@@ -60,8 +55,10 @@
     private Cleanups() {
     }
 
-    public static List<Formatter> getAvailableFormatters() {
-        return Collections.unmodifiableList(availableFormatters);
+    public static List<Formatter> getBuiltInFormatters() {
+        List<Formatter> availableFormatters = new ArrayList<>();
```

Comment:
Why not just new ArrayList(formatters.getAll())?

Reply:
Good point, done.